src/Sulu/Bundle/SecurityBundle/Entity/User.php

```diff
@@ -313,7 +313,7 @@ public function getPrivateKey()
     /**
      * Removes the password of the user.
      */
-    public function eraseCredentials()
+    public function eraseCredentials(): void
```

Comment:
@chirimoya should we cherry pick the bc breaks already into 2.6 branch. Think it would be better to make the bc breaks in the minor instead of a patch release?

Reply:
👍